### PR TITLE
feat: support grace_period_seconds in terminate_pods action

### LINKED ISSUE #22: Include grace_period_seconds to the terminate_pods action
It would be interesting to add `grace_period_seconds` paramater to [the V1DeleteOptions at chaosk8s/pod/actions.py](https://github.com/chaostoolkit/chaostoolkit-kubernetes/blob/master/chaosk8s/pod/actions.py#L82) so we can choose whether we want to "kill" a pod (terminate it as soon as possible, i.e. grace_period_seconds=0) instead of gracefully terminating it.

### PR DESCRIPTION
**Pull request by palmerabollo**
Support a new `grace_period_seconds` parameter in the `terminate_pods` action to configure the period before a pod is allowed to be forcefully killed. See also https://kubernetes.io/docs/concepts/workloads/pods/pod/#termination-of-pods

Resolves #22

Signed-off-by: Guido García <guido.garciabernardo@telefonica.com>

### CODE REVIEW

File: chaosk8s/pod/actions.py [hunk 1]
```diff
@@ -79,7 +84,10 @@ def terminate_pods(label_selector: str = None, name_pattern: str = None,
     logger.debug("Picked pods '{p}' to be terminated".format(
         p=",".join([po.metadata.name for po in pods])))
 
-    body = client.V1DeleteOptions()
+    if grace_period >= 0:
```
Comment (Lawouach) [inline chaosk8s/pod/actions.py]: I wonder if we could collapse those conditionals into a single line? How does the API react to grace_period=-1 as a delete option? Or None?

Comment (palmerabollo) [reply chaosk8s/pod/actions.py]: Not sure. I've just tried a `kubectl delete pod xxxx --grace-period=-1` and the operation works but I can't say what Kubernetes is doing (ignoring it vs using zero vs another) under the hood.

Comment (palmerabollo) [reply chaosk8s/pod/actions.py]: LGTM. Python is not my bread and butter as you might have noticed. Any feedback is welcome.

Comment (Lawouach) [reply chaosk8s/pod/actions.py]: I actually, I thought your changes looked clean so I had assumed you coded in Python :)

**Pull request merged**